packages/builders/plugins/jest/resolver.spec.ts

```diff
@@ -0,0 +1,36 @@
+jest.mock('jest-resolve/build/default_resolver');
+import jestResolve from 'jest-resolve/build/default_resolver';
+import * as _resolve from './resolver';
+import * as ts from 'typescript';
+
+const resolve = _resolve as Function;
+
+describe('Jest resolver', () => {
```

Comment:
I don't think there is much value in this spec. You are basically testing that try/catch works.

I think having e2e tests exercising the jest builder and hence the resolve is good enough. Having integration tests for the resolved would have been good too.

I'd delete it

Reply:
e2e tests will definitely be the real test here. Let me remove these as it's a false sense of security. Let's wait on the schematics will generate the configs so I don't have to write it twice.